### ISSUE TITLE
undeline links in content

### ISSUE THREAD

**Issue by 00travelgirl00** [WordPress/twentynineteen#15]
Links within content should be underlines, otherwise they are hard to see for people with colorblindness. 

<img width="837" alt="bildschirmfoto 2018-10-16 um 16 27 05 kopie" src="https://user-images.githubusercontent.com/21318377/47023815-bfd0d500-d160-11e8-9db9-c04617a3a76a.png">
<img width="837" alt="bildschirmfoto 2018-10-16 um 16 27 05" src="https://user-images.githubusercontent.com/21318377/47023816-c0696b80-d160-11e8-9952-490be5f94654.png">
<img width="945" alt="bildschirmfoto 2018-10-16 um 16 28 13" src="https://user-images.githubusercontent.com/21318377/47023818-c0696b80-d160-11e8-9f49-921f20ad6434.png">

**Comment by joedolson** [comment]
Note that this is a requirement for the accessibility-ready tag on WordPress.org: https://make.wordpress.org/themes/handbook/review/accessibility/required/#content-links

**Comment by melchoyce** [comment]
Could be a fun opportunity to play with some styled borders — dashes, a thicker line, a double line, etc.

**Comment by audrasjb** [comment]
Yes, good catch. 
And I'd say we should also underline links in comments area (which are mostly often ignored but that can happens).

**Comment by joedolson** [comment]
I'd argue that comments **are** content; but if you think I should clarify that thinking in the theme accessibility guidelines, I'll do that.

**Comment by audrasjb** [comment]
You are right, however linksin comments are often ignored because you can’t publish links in comment by default. This issue have been resolved in Twenty Seventeen almost a year after Twenty Seventeen was released ;)

**Comment by bamadesigner** [comment]
For WCAG 2.0 AA reference: https://www.w3.org/WAI/WCAG21/quickref/#use-of-color

**Comment by nielslange** [comment]
I just created #186 and added screenshots of the border styles `dotted`, `dashed`, `solid` and `double`.

CC: @melchoyce
